code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -979,7 +979,7 @@ GLOBAL_DATUM_INIT(sdql2_vv_statobj, /obj/effect/statclick/sdql2_vv_all, new(null
 	return querys
 
 /proc/SDQL_testout(list/query_tree, indent = 0)
-	var/static/whitespace = "&nbsp;&nbsp;&nbsp; "
+	var/static/whitespace = STATIC_INIT("&nbsp;&nbsp;&nbsp; ")
```

Comment:
similarly redundant? unless it's somehow worth while to hang a ref to a string permanently?

Reply:
Yes, but out of scope